target-gen/src/generate.rs

```diff
@@ -131,6 +131,7 @@ where
             .iter()
             .map(create_core)
             .collect::<Result<Vec<_>>>()?;
+        println!("{:?}", cores.len());
```

Comment:
Please add an explanation, and don't just print a number without context.

Reply:
Hmm this definitely got in here by accident. Should not have been.